src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -339,7 +340,11 @@ protected function describeContainerDefinition(Definition $definition, array $op
                 if ($argument instanceof Reference) {
                     $argumentsInformation[] = sprintf('Service(%s)', (string) $argument);
                 } elseif ($argument instanceof IteratorArgument) {
-                    $argumentsInformation[] = sprintf('Iterator (%d element(s))', \count($argument->getValues()));
+                    if ($argument instanceof TaggedIteratorArgument) {
+                        $argumentsInformation[] = sprintf('Tagged Iterator for "%s" %s', $argument->getTag(), !$options['is_debug'] ? sprintf('(%d element(s))', \count($argument->getValues())) : '');
+                    } else {
+                        $argumentsInformation[] = $options['is_debug'] ? 'Iterator' : sprintf('Iterator (%d element(s))', \count($argument->getValues()));
```

Comment:
We maybe want to be consistent wether we handle the debug or the non-debug case first.

Reply:
adjusted

Reply:
displaying useful info is more important than "consistency" here to me

Reply:
displaying useful info is more important than "consistency" here to me